sync/src/orphan_block_pool.rs

```diff
@@ -118,6 +145,14 @@ impl OrphanBlockPool {
         self.inner.read().get_block(hash)
     }
 
+    pub fn get_block_by_parent(&self, parent_hash: &packed::Byte32) -> Option<core::BlockView> {
```

Comment:
delete this unnecessary function

Reply:
agree! less code, same functionality, sounds good.